igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/config/TravisConfig.java

```diff
@@ -123,27 +126,39 @@ public Map<String, TravisService> travisMasters(
     return travisMasters;
   }
 
-  public static TravisClient travisClient(String address, int timeout, ObjectMapper objectMapper) {
+  public static TravisClient travisClient(
+      String address,
+      int timeout,
+      ObjectMapper objectMapper,
+      OkHttp3ClientConfiguration okHttpClientConfig) {
     OkHttpClient client =
-        new OkHttpClient.Builder().readTimeout(timeout, TimeUnit.MILLISECONDS).build();
+        okHttpClientConfig
+            .createForRetrofit2()
+            .readTimeout(timeout, TimeUnit.MILLISECONDS)
+            .addInterceptor(new TravisHeader())
+            .build();
+    new OkHttpClient.Builder().readTimeout(timeout, TimeUnit.MILLISECONDS).build();
 
-    return new RestAdapter.Builder()
-        .setEndpoint(Endpoints.newFixedEndpoint(address))
-        .setRequestInterceptor(new TravisHeader())
-        .setClient(new Ok3Client(client))
-        .setConverter(new JacksonConverter(objectMapper))
-        .setLog(new Slf4jRetrofitLogger(TravisClient.class))
-        .setErrorHandler(SpinnakerRetrofitErrorHandler.getInstance())
+    return new Retrofit.Builder()
+        .baseUrl(address)
```

Comment:
Can you refer [this](https://github.com/spinnaker/echo/pull/1476) and make sure the address doesn't end up in `baseUrl must end in /` error. And this is applicable to all such retrofit2 client definitions.